src/tests/ftest/io/romio.py

```diff
@@ -131,7 +121,7 @@ def test_romio(self):
                              "failed to destroy pool"]
 
             for line in searchfile:
-                for i in xrange(len(error_message)):
+                for i in range(len(error_message)):
```

Comment:
(pylint-consider-using-enumerate) Consider using enumerate instead of iterating with range and len